test/unit/controllers/Cid_EntryController.test.js

```diff
@@ -211,6 +213,16 @@ describe('UNIT Cid_EntryController', () => {
     });
   });
 
+  describe('publishCentreUpdates', () => {
+    it('should eventually update Centres with cid_received_date', (done) =>
+      controller.publishCentreUpdates().then(() => {
+        expect(Centres.update).to.have.been.calledOnce;
```

Comment:
you can do `calledOnceWith` and match
or you can use the chai things to .has.a.thing.that.contains (or similar syntax, can't remember off the top of my head)